backend/app/services/rag/pipeline/pandoc.py

```
@@ -0,0 +1,288 @@
+            Markdown text content
+
+        Raises:
+            PandocConversionError: If conversion fails
+        """
+        ext = file_extension.lower()
+        input_format = self.INPUT_FORMAT_MAP.get(ext)
+
+        if not input_format:
+            raise PandocConversionError(
+                f"Unsupported file extension for Pandoc: {file_extension}"
+            )
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)